strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/domain/PypiMetadataDao.java

```diff
@@ -0,0 +1,95 @@
+package org.carlspring.strongbox.domain;
+
+import javax.inject.Qualifier;
+import java.lang.annotation.*;
+
+public class PypiMetadataDao {
+
+    @PypiMetadataKey(name = "Metadata-Version")
+    private String metadataVersion;
+
+    @PypiMetadataKey
+    private String name;
+
+    @PypiMetadataKey(name = "Version")
+    private String version;
+
+    @PypiMetadataKey(name = "Summary")
+    private String summary;
+
+    @PypiMetadataKey(name = "Home-page")
+    private String homePage;
+
+    @PypiMetadataKey(name = "Author")
+    private String author;
+
+    @PypiMetadataKey(name = "Author-email")
+    private String authorEmail;
+
+    @PypiMetadataKey(name = "License")
+    private String license;
+
+    @PypiMetadataKey(name = "Description-Content-Type")
+    private String descriptionContentType;
+
+    @PypiMetadataKey(name = "Description")
+    private String description;
+
+    @PypiMetadataKey(name = "Platform")
+    private String platform;
+
+
+    public String getMetadataVersion() {
+        return metadataVersion;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getVersion() {
+        return version;
+    }
+
+    public String getSummary() {
+        return summary;
+    }
+
+    public String getHomePage() {
+        return homePage;
+    }
+
+    public String getAuthor() {
+        return author;
+    }
+
+    public String getAuthorEmail() {
+        return authorEmail;
+    }
+
+    public String getLicense() {
+        return license;
+    }
+
+    public String getDescriptionContentType() {
+        return descriptionContentType;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public String getPlatform() {
+        return platform;
+    }
+
+    @Retention(RetentionPolicy.RUNTIME)
+    @Target(ElementType.FIELD)
+    @Qualifier
+    @Documented
+    public static @interface PypiMetadataKey {
```

Comment:
I don't quite think we'll actually need this class. (I get why one might need it, but in our scenario I don't think this would be applicable).

Reply:
Thanks for your comment! My question then is, how can we annotate fields with annotation @PypiMetadataKey without creating custom @interface? Thank you for your time :)

Reply:
I may have gotten confused. Sorry!

Could I ask you to make this a separate (non-nested) class? Thanks! :)